src/Semantics.js

```diff
@@ -196,6 +196,41 @@ Semantics.prototype.checkActionDicts = function() {
   }
 };
 
+Semantics.prototype.toRecipe = function() {
+  var sb = new common.StringBuffer();
```

Comment:
Now... yes! :-p (added a `console.error(all)` and `2> /dev/null`)
But I was thinking that it might make a difference in a bigger context - yes.
(https://jsperf.com/string-concat-vs-array-join-10000/)